Source/ChocolateyGui/ChocolateyGui.csproj

```diff
@@ -495,6 +495,7 @@
       <Generator>PublicResXFileCodeGenerator</Generator>
       <LastGenOutput>Resources.de.Designer.cs</LastGenOutput>
     </EmbeddedResource>
+    <EmbeddedResource Include="Properties\Resources.sv.resx" />

```

Comment:
I think you're right @mwallner. Why doesn't this look like the other ones? @magol Could you update this entry to look like the other two resource file entries?